chef/cookbooks/ceph/recipes/radosgw_ha.rb

```diff
@@ -33,29 +33,37 @@
 
 transaction_objects = []
 
-service_name = "ceph-radosgw"
+# stolen from radosgw.rb, sort of; we're using just the instance name
+# for the pacemaker primitive, because rgw.hostname is enough to make
+# it unique, and anyway we can't prefix it with "ceph-radosgw@" because
+# '@' is invalid in pacemaker primitive IDs...
+rgw_hostname = get_ceph_client_name(node)
+service_name = "rgw.#{rgw_hostname}"
+
 pacemaker_primitive service_name do
-  agent node[:ceph][:ha][:radosgw][:agent]
+  # ...but we still need the full "ceph-radosgw@..." form here for systemd.
+  agent "systemd:ceph-radosgw@#{service_name}"
   op node[:ceph][:ha][:radosgw][:op]
   action :update
-  only_if { CrowbarPacemakerHelper.is_cluster_founder?(node) }
 end
 transaction_objects << "pacemaker_primitive[#{service_name}]"
 
-clone_name = "cl-#{service_name}"
-pacemaker_clone clone_name do
-  rsc service_name
-  meta ({ "clone-max" => CrowbarPacemakerHelper.num_corosync_nodes(node) })
+location_constraint = "l-#{service_name}"
+pacemaker_location location_constraint do
+  # I tried inf: for this node, but pacemaker would still try to start the
+  # resource on other nodes when this node went down, hence the reversed
```

Comment:
Weird, I would expect it to be symmetric, i.e. for `inf` to work fine.  Can you check with the SLE HA team on this?